pkg/mutate/squashfs.go

```diff
@@ -45,13 +46,29 @@ func OptSquashfsLayerConverter(converter string) SquashfsConverterOpt {
 
 var errSquashfsConverterNotSupported = errors.New("squashfs converter not supported")
 
+// OptNoConvertWhiteout is set to skip the default conversion of whiteout /
+// opaque markers from AUFS to OverlayFS format.
+func OptNoConvertWhiteout(b bool) SquashfsConverterOpt {
+	return func(c *squashfsConverter) error {
```

Comment:
Let's namespace this so it's clear it's related to `Squashfs`. And maybe "skip" instead of "no" reads a little cleaner?

```suggestion
// OptSquashfsSkipWhiteoutConversion is set to skip conversion of whiteout/opaque markers from AUFS to
// OverlayFS format.
func OptSquashfsSkipWhiteoutConversion(b bool) SquashfsConverterOpt {
```